email_campaigns/thanks-mentor-email.md

```diff
@@ -0,0 +1,36 @@
+{{ name }},
+
+# Thanks for mentoring at the Mentored Sprint for Diverse Beginners at PyCon US 2019!
+
+<insert photo?> (not sure if possile)
+
+We want to take the time to thank each and everyone of you for joining us, for being
+excellent, and for contributing to this amazing community. You've inspired many new
+open source contributors, and the Python community is better because of you
+💜
+
+This is our first time organizing the mentored sprint, and numerous people
+have asked when will we do it again! Before we can get to that, we'd like to hear
+about your experience as the project mentor.
+
+We are including an anonymous survey to hear your thoughts on what we can do even better next time. 
+
+Give Feedback: http://bit.ly/mentors-feedback-mentoredsprints
```

Comment:
remove space line and maybe *Feedback Form* instead of Give Feedback?

Reply:
@Nikoleta-v3  and @Mariatta What about "Give your feedback" instead? I think that's an actionable phrase